.github/workflows/pytest_testing.yml

```diff
@@ -0,0 +1,30 @@
+name: Pytest Testing
+
+on:
+  pull_request:
+    types: [opened,reopened,synchronize]
+  push:
+    branches:    
+      - 'main'
+      - 'releases/**'
+
+jobs:
+  test:
+    name: pytest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+      - name: Setup Environment
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[jwst,hst,test]
+          mkdir $HOME/crds_cache
+          echo "CRDS_PATH=$HOME/crds_cache" >> $GITHUB_ENV
+          echo "CRDS_SERVER_URL=https://jwst-crds.stsci.edu" >> $GITHUB_ENV
+      - name: Run tests
+        run: pytest tests --cov=. --cov-report xml
+#      - name: Upload coverage reports to Codecov with GitHub Action
```

Comment:
I think you can uncomment these lines now.  Let me know if there's anything else I need to do.